vnext/Microsoft.ReactNative/Utils/ValueUtils.cpp

```diff
@@ -86,6 +86,10 @@ xaml::Media::Brush BrushFromColorObject(winrt::hstring resourceName) {
 
   winrt::IInspectable resource{winrt::Application::Current().Resources().Lookup(winrt::box_value(resourceName))};
 
+  if (auto color = resource.try_as<ui::Color>()) {
```

Comment:
Nit: Should we do the if check on brush, and return that, assuming that the brush case is heavier usage, so we save a QI?